public/scss/main.scss

```diff
@@ -191,9 +191,10 @@ body {
 }
 
 .verticalLine {
-  border-right: thin dotted $yellow;
-  height: 35px;
-  width:1px;
+    border-right: 3px solid #FDCC06;
```

Comment:
Should probably keep using the $yellow variable - it's the exact same color (https://github.com/helloGov/singleAction/blob/master/public/scss/_colors.scss#L1). Then if that color definition were to change this would automatically get updated too.